client/src/ui/PageFooter.tsx

```diff
@@ -0,0 +1,172 @@
+"use client";
+import Link from "next/link";
+import Image from "next/image";
+import { Typography, Divider } from "@mui/material";
+
+/*
+This Page Footer will be responisble for rendering the footer of all pages in the project
+Except for the Login Page and Sign Up Page
+*/
+
+export const PageFooter = () => {
+  return (
+    <footer
+      style={{
+        backgroundColor: "#f9f9fc",
+        padding: "20px 40px",
+        display: "flex",
+        justifyContent: "space-evenly",
+        flexWrap: "wrap",
+      }}
+    >
+      {/* Left Section */}
+      <div
+        style={{
+          display: "flex",
+          flexWrap: "nowrap",
+        }}
+      >
+        <Link
+          href="#"
+          style={{ margin: "0 40px", color: "#000", textDecoration: "none" }}
+        >
+          <Typography>Explore</Typography>
+        </Link>
+        <Link
+          href="#"
+          style={{ margin: "0 40px", color: "#000", textDecoration: "none" }}
+        >
+          <Typography>About Us</Typography>
+        </Link>
+        <Link
+          href="#"
+          style={{ margin: "0 40px", color: "#000", textDecoration: "none" }}
+        >
+          <Typography>Contact</Typography>
+        </Link>
+      </div>
+
+      {/* Center Section */}
+      <div
+        style={{
+          display: "flex",
+          alignItems: "center",
+          textAlign: "center",
+          position: "relative",
+          right: "15px",
+        }}
+      >
+        <Divider
+          style={{
+            flexGrow: 1,
+            backgroundColor: "#ccc",
+            width: "20px",
+            height: "1px",
+            margin: "0 20px",
+          }}
+        />
+        <Typography
+          variant="h4"
+          style={{
+            fontWeight: "bold",
+            fontSize: "1.25rem",
+            textAlign: "center",
+          }}
+        >
+          SqueakPeek
+        </Typography>
+        <Divider
+          style={{
+            flexGrow: 1,
+            backgroundColor: "#ccc",
+            width: "20px",
+            height: "1px",
+            margin: "0 20px",
+          }}
+        />
+      </div>
+
+      {/* Right Section */}
+      <div
+        style={{
+          display: "flex",
+          alignItems: "center",
+        }}
+      >
+        <Link href="#">
```

Comment:
The Images component are redundant which might make it hard to maintain in the future. It also makes it hard to change most of them later. One way to remove redundancy is by mapping through the array of the metadata of each, and rendering a component. Ex:

const images = [
{
src: "/metaImage",
alt: "Icon of meta"
route: "/{route to meta page"
},
...
]

`images.map((image) => { <Link href={image.route}> <Image src={image.src} alt={image.alt} />}`

This removes redundancy and makes it easier to change and maintain in the future. This can also apply to the left section of the footer. You can take a look at the NavBar I created as an example